Libraries/Components/Keyboard/KeyboardAvoidingView.js

```diff
@@ -191,7 +191,7 @@ class KeyboardAvoidingView extends React.Component<Props, State> {
               style={StyleSheet.compose(
                 contentContainerStyle,
                 {
-                  bottom: bottomHeight,
+                  bottom: bottomHeight + contentContainerStyle.bottom,
```

Comment:
Tests are failing because this isn't accounting for the null case for `contentContainerStyle` and `bottom` property

Reply:
Also this needs to verify  that `contentContainerStyle.bottom` is a number value before adding

Reply:
Should also add: 
`bottomHeight + style != null ? style.paddingBottom : 0,`
to line 208